java-frontend/src/main/java/org/sonar/java/Measurer.java

```diff
@@ -111,12 +112,20 @@ public void scanFile(JavaFileScannerContext context) {
 
     RangeDistributionBuilder fileComplexityDistribution = new RangeDistributionBuilder(LIMITS_COMPLEXITY_FILES);
     saveMetricOnFile(CoreMetrics.FILE_COMPLEXITY_DISTRIBUTION, fileComplexityDistribution.add(fileComplexity).build());
+
+    if (isSonarQubeGreaterThanOrEqualTo63()) {
```

Comment:
It seems that this version checking is needed in multiple places in the code, maybe it is worth to have single utility class handling these checks.